### PR TITLE
Con 2528 enhance confirmation page

### PR DESCRIPTION
**Pull request by kola-samuel**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs [hunk 1]
```diff
@@ -175,7 +175,7 @@ public async Task<IActionResult> SendRequestNewTrainingProvider(SendNewTrainingP
                 {
                     var apiRequest = await _modelMapper.Map<CreateChangeOfPartyRequestRequest>(request);
                     await _commitmentsApiClient.CreateChangeOfPartyRequest(request.ApprenticeshipId, apiRequest);
-                    return RedirectToRoute(RouteNames.ChangeProviderRequestedConfirmation, new { request.AccountHashedId, request.ApprenticeshipHashedId, request.ProviderId });
+                    return RedirectToRoute(RouteNames.ChangeProviderRequestedConfirmation, new { request.AccountHashedId, request.ApprenticeshipHashedId, request.ProviderId , ProviderAddDetails = true});
```
Comment (hkf-tech) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs:178]: my be add a test to check ProviderAddDetails value is getting set as true.

**Review by hkf-tech — APPROVED** [review]

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs (hunk 1 above)
Comment (kola-samuel) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs:178]: There are a number of test already I will look into it thanks

File: src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/WhenPostingSendRequestNewTrainingProviderTests.cs [hunk 2]
```diff
@@ -146,5 +156,24 @@ public void VerifyRedirectToError(IActionResult actionResult)
             Assert.AreEqual("Error", redirectResult.ControllerName);
             Assert.AreEqual("Error", redirectResult.ActionName);
         }
+
+        public void VerifyProviderAddDetailsIsSetToTrue(IActionResult result)
```
Comment (hkf-tech) [inline src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/WhenPostingSendRequestNewTrainingProviderTests.cs:160]: I was looking for another example fo this and found the following on WhenCallingMessage.cs, the following looks a bit more concise and simple, but happy to discuss

   var redirect = result.VerifyReturnsRedirectToActionResult();
            Assert.AreEqual(redirect.ActionName, "Finished");
            Assert.AreEqual(redirect.RouteValues["AccountHashedId"], model.AccountHashedId);
            Assert.AreEqual(redirect.RouteValues["CohortReference"], createCohortResponse.CohortReference);

**Review by hkf-tech — CHANGES_REQUESTED** [review]

**Review by hkf-tech — APPROVED** [review]

**Comment by kola-samuel** [comment]
The code has been merged in the FeatureBranch_CoP-Employer-lead

**Pull request closed without merging**